engine/dlib/src/cs/dmsdk/dlib/configfile_gen.cs

```diff
@@ -15,36 +15,103 @@
 // CONDITIONS OF ANY KIND, either express or implied. See the License for the
 // specific language governing permissions and limitations under the License.
 
+
+/*# SDK ConfigFile API documentation
+ *
+ * Configuration file access functions.
+ * The configuration file is compiled version of the [file:game.project] file.
+ *
+ * @document
+ * @language C#
+ * @name ConfigFile
+ * @namespace Dlib
+ * @path engine/dlib/src/cs/dmsdk/dlib/configfile_gen.cs
+ */
+
```

Comment:
We now have access to the source documentation, when generating source code for other languages.